packages/core/src/diff/changes/change.ts

```diff
@@ -73,11 +73,578 @@ export enum CriticalityLevel {
 export interface Criticality {
   level: CriticalityLevel;
   reason?: string;
+  /** Is `true` if the criticality is safe, because of the usage determined via the `checkUsage` config option. */
+  isSafeBasedOnUsage?: boolean;
```

Comment:
is it a new thing? should we document it?

Reply:
maybe later